test/libsolidity/smtCheckerTests/options/engine_none.sol

```diff
@@ -0,0 +1,7 @@
+contract C {
+	function f(uint x) public pure {
+		assert(x > 0);
+	}
+}
```

Comment:
We recommend 4-space idents for Solidity in our style guideline so I think we should use that in our tests too (at least the newly written ones).

Reply:
Done